lib/slosilo/adapters/sequel_adapter.rb

```diff
@@ -49,6 +49,13 @@ def each
         end
       end
 
+      def recalculate_fingerprints
```

Comment:
This method can now be invoked outside of a transaction...

Reply:
What is the concern of someone being able to invoke this outside of a transaction? Currently, our only other way to invoke an update on the fingerprint column is by deleting it and then calling `migrate!`.

Reply:
Now we can trigger this m.update (which is a write command, more hazard than read, see `each`)
without checking `fingerprint_in_db?` and without transaction, thus a mix of fingerprint is now a valid option.

Reply:
Good catch @uCatu.

Reply:
@uCatu is your concern that we will have a mix of `new` and `old` fingerprints in the table at the same time because we are updating them one by one? If so would a batch update solve your concern?